_scss/components/_sidenav.scss

```
@@ -1,5 +1,13 @@
+@use "uswds-core" as *;
+@use "theme-variables" as *;
+
+.font-sans-2xs {
+  font-size: 0.88rem;
+}
+
 .usa-sidenav {
   @include u-border-bottom("base");
+  font-size: .88rem;
```

Let's add leading zero so it's consistent to the reference above (line 5).

```suggestion
  font-size: 0.88rem;
```

---

Good point